tests/index.js

```diff
@@ -195,6 +214,100 @@ module.exports = {
+      .getLocationInView(eHeader, (result) => {
+        browser.assert.equal(typeof result, 'object');
+        browser.assert.equal(result.status, 0);
+        browser.assert.equal(result.value.y, 0, 'anchor header should be at the top of the page view');
+      });
+
+  },
+  'when resizing window to mobile view, sidebar should collapse': function (browser){
+    const stepOptions = getStepOptions('scrollToAnchor', browser);
+
+    browser
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .waitForElementVisible(rootTag, 1000)
+      .resizeWindow(mobileSize.x, mobileSize.y)    // iPhone 6 specs
+      .assert
+      .cssProperty('.doc-sidebar', 'transform', 'matrix(1, 0, 0, 1, -230, 0)');  // TODO: Check if better way to this
+  },
+  'when resizing window to mobile view, burger menu icon is displayed': function (browser){
+    const stepOptions = getStepOptions('scrollToAnchor', browser);
+
+    browser
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .waitForElementVisible(rootTag, 1000)
+      .resizeWindow(mobileSize.x, mobileSize.y)    // iPhone 6 specs
+      .assert
+      .visible('.doc-navbar .doc-sidebar-toggle', 'burger menu icon should be visible on mobile');
+  },
+  'when resizing window to mobile view, clicking on burger menu should open sidebar': function (browser){
+    const stepOptions = getStepOptions('scrollToAnchor', browser);
+
+    browser
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .waitForElementVisible(rootTag, 1000)
+      .resizeWindow(mobileSize.x, mobileSize.y)    // iPhone 6 specs
+      .click('.doc-navbar .doc-sidebar-toggle')
+      .pause(1000)
+      .assert
+      .cssProperty('.doc-sidebar', 'transform', 'matrix(1, 0, 0, 1, 0, 0)');
+  },
+  'search results page should display total number of results': function (browser){
+    const stepOptions = getStepOptions('searchResultItems', browser);
+
+    browser
+      .resizeWindow(desktopSize.x, desktopSize.y)    // iPhone 6 specs
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .waitForElementVisible(rootTag, 1000)
+      .setValue(stepOptions.searchInputSelector, stepOptions.searchQuery)
+      .pause(1000)
+      .elements('css selector', stepOptions.searchResultItemSelector, function (results){
+        browser
+          .expect
+          .element(stepOptions.searchResultsTitleSelector)
+          .text
+          .to
+          .match(new RegExp('^' + results.value.length));
+      });
+  },
+  'search results display message for 0 results': function (browser){
+    const stepOptions = getStepOptions('searchResultItems', browser);
+
+    browser
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .waitForElementVisible(rootTag, 1000)
+      .setValue(stepOptions.searchInputSelector, stepOptions.searchQuery + '!@#$%^&*()')
+      .pause(1000)
+      .expect
+      .element('.doc-search-results__title + p')
+      .text
+      .to
+      .match(/\w+/);
```

Comment:
I will try to test that text match. Pseudo-code : `match(new RegExp('^There are no results for "${usedQuery}".*'))`